Tests/ArcGISToolkitTests/LocationButtonTests.swift

```diff
@@ -0,0 +1,148 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+@testable import ArcGISToolkit
+import Testing
+
+@Suite("LocationButton Tests")
+struct LocationButtonTests {
+    @Test
+    @MainActor
+    func testInit() {
```

Comment:
```suggestion
    func init() {
```

Can the test prefix be removed since these are since Swift Testing is used?

Same suggestion applies elsewhere.